doc/api/net.md

```diff
@@ -39,9 +39,10 @@ socket outside of these abstractions, the user will need to remove it. The same
 applies when a Node.js API creates a Unix domain socket but the program then
 crashes. In short, a Unix domain socket will be visible in the file system and
 will persist until unlinked. On Linux, You can use Unix abstract socket by adding
-`\0` to the beginning of the path, such as `\0abstract`. The path to the Unix
-abstract socket is not visible in the file system and it will disappear automatically
-when all open references to the socket are closed.
+`\0` to the beginning of the path, such as `\0abstract` (Only available in
+Node.js >= v20.8.0). The path to the Unix abstract socket is not visible in the
```

Comment:
It was available before v20.4.0 (per https://github.com/nodejs/node/pull/48618 and https://github.com/nodejs/node/issues/49656). I'd find the loss of the support to be a bug rather than a new feature introduced in v20.8.0.

Reply:
version < v20.4.0: we can bind '\0xxx' successfully but in fact the socket just bind to '\0' instead of '\0xxx'.
v20.4.0 =< version < v20.8.0: we can bind to '\0xxx' any more.
>= v20.8.0: we can bind to '\0xxx' or '\0yyy'.

So i think we really support Unix domain socket until version >= v20.8.0 ?

Reply:
In this sense, I would find listing the changes with the YAML frontmatter could be a more structured way to describe change history.

```
<!-- YAML
changes:
  - version: v20.8.0
    pr-url: https://github.com/nodejs/node/pull/49667
    description: Support binding to abstract unix domain socket path like `\0abstract`.
-->
```

Reply:
Do you mean we should add the description in the change history?